WHO.js

```diff
@@ -148,12 +145,42 @@ function scrape(doc, url) {
 		item.complete();
 	});
 
-	translator.getTranslatorObject(function(trans) {
+	translator.getTranslatorObject(function (trans) {
 		trans.itemType = type;
 		trans.doWeb(doc, url);
 	});
 }
 
+function detectSearch(item) {
+	return item.ISBN && ZU.cleanISBN(item.ISBN).startsWith('978924');
+}
+
+function doSearch(item) {
+	let ISBN = ZU.cleanISBN(item.ISBN);
+	let url = `https://apps.who.int/iris/discover/export?format=refman&list=discover&rpp=10&etal=0&query=${ISBN}&group_by=none&page=1`;
+	ZU.doGet(url, function (risText) {
```

Comment:
At least from curl, it looks like this returns an empty 200 response in the case of an unknown ISBN. Not sure what the RIS translator does on an empty string, but maybe cleaner to test for an empty `risText` here and bail.

Reply:
The RIS translator just returns no items when it gets an empty string, so I don't think bailing will do anything but save us a little time loading the translator. But there's no reason not to, so I'll add it.